gems/smithy/spec/spec_helper.rb

```diff
@@ -56,7 +69,8 @@ def create_plan(modules, model, type, options)
       plan_options = {
         gem_name: options[:gem_name] || Smithy::Tools::Namespace.gem_name_from_namespaces(modules),
         gem_version: options[:gem_version] || '1.0.0',
-        destination_root: options[:destination_root] || Dir.mktmpdir
+        destination_root: options[:destination_root] || Dir.mktmpdir,
+        skip_polishes: options.fetch(:skip_polishes, true)
```

Comment:
Can you use || or make all of these fetch?

Reply:
|| doesn't work well for booleans, so switched to using fetch for everything.